scripts/countlines.sh

```diff
@@ -2,32 +2,34 @@
 
 # Count lines in file given in argument
 
+error_arg_is_directory='An argument must be a regular file not directory'
+error_one_arg='Must be one argument'
+error_arg_is_not_path='An argument is not a path'
```

Comment:
change message 'An argument is not a file path'

Reply:
+